.changeset/big-eels-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/gatsby-theme-doctocat": major
```

Comment:
⚠️ Should this be a breaking change or nah?

Reply:
Hmm, yes? I'm not sure